build.sbt

```diff
@@ -165,6 +166,8 @@ lazy val docs = project("docs")
         val v = if (isSnapshot.value) "current" else version.value
         s"http://doc.akka.io/japi/akka-http/$v"
       },
+      "algolia.docsearch.api_key" -> "0ccbb8bf5148554a406fbf07df0a93b9",
+      "algolia.docsearch.index_name" -> "akka-http",
```

Comment:
something for google analytics also https://github.com/akka/akka/pull/22942/files